ci-operator/step-registry/rosa/sts/proxy/rosa-sts-proxy-workflow.yaml

```diff
@@ -0,0 +1,25 @@
+workflow:
+  as: rosa-sts-proxy
```

Comment:
Let us define the workflow under `rosa/aws/sts` with the defined chains.

Reply:
so should I move the workflow over to rosa/aws/sts/provision/proxy?

Reply:
yes, to `rosa/aws/sts/provision/proxy`

Reply:
latest commit has the chain and workflow moved as requested, I left the env vars in the workflow as I couldn't get them accepted in the chain despite the docs saying it should be allowed.